modules/document_repository/php/document_repository.class.inc

```diff
@@ -78,9 +78,9 @@ class Document_Repository extends \NDB_Menu_Filter
                           'v.Data_dir',
                           'v.Date_uploaded',
                          );
-        $openAccordion = isset($_GET['openAccordion']) ?? '';
+        $openAccordion = isset($_GET['openAccordion']) ?? false;
```

Comment:
The correct idiom here would be 
`$openAccordion = $_GET['openAccordion'] ?? false;`
Same with the one below.